<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>alarm_plugin::EditAlarmDialog</class>
 <widget class="QDialog" name="alarm_plugin::EditAlarmDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>233</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Alarm Settings</string>
  </property>
  <layout class="QGridLayout" name="dialog_layout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="time_label">
     <property name="text">
      <string>Time:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QTimeEdit" name="time_edit"/>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>190</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3">
    <widget class="QCheckBox" name="is_enabled_checkbox">
     <property name="text">
      <string>enable this alarm</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QGroupBox" name="days_group">
     <property name="title">
      <string>Days</string>
     </property>
     <layout class="QHBoxLayout" name="days_layout">
      <item>
       <widget class="QCheckBox" name="day_1">
        <property name="text">
         <string notr="true">&lt;d1&gt;</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="day_2">
        <property name="text">
         <string notr="true">&lt;d2&gt;</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="day_3">
        <property name="text">
         <string notr="true">&lt;d3&gt;</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="day_4">
        <property name="text">
         <string notr="true">&lt;d4&gt;</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="day_5">
        <property name="text">
         <string notr="true">&lt;d5&gt;</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="day_6">
        <property name="text">
         <string notr="true">&lt;d6&gt;</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="day_7">
        <property name="text">
         <string notr="true">&lt;d7&gt;</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QGroupBox" name="sound_group">
     <property name="title">
      <string>Sound</string>
     </property>
     <layout class="QGridLayout" name="sound_layout">
      <item row="0" column="0">
       <widget class="QLabel" name="file_label">
        <property name="text">
         <string>File:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QLineEdit" name="filepath_edit"/>
      </item>
      <item row="0" column="4">
       <widget class="QToolButton" name="browse_btn">
        <property name="icon">
         <iconset resource="../alarm.qrc">
          <normaloff>:/alarm/browse.svg.p</normaloff>:/alarm/browse.svg.p</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="volume_label">
        <property name="text">
         <string>Volume:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSlider" name="volume_slider">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>75</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="volume_value">
        <property name="text">
         <string notr="true">75</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="percent_label">
        <property name="text">
         <string notr="true">%</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QToolButton" name="preview_btn">
        <property name="icon">
         <iconset resource="../alarm.qrc">
          <normaloff>:/alarm/play.svg.p</normaloff>:/alarm/play.svg.p</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../alarm.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>alarm_plugin::EditAlarmDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>alarm_plugin::EditAlarmDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>volume_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>volume_value</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
